indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -263,7 +267,7 @@
             if (categoriesWithSubcategories.disabled && !categoriesWithEvents.disabled && category.deep_category_count) {
                 $button.addClass('disabled').attr('title', categoriesWithSubcategories.message);
             }
-            if (categoriesWithEvents.disabled && !categoriesWithSubcategories.disabled && category.has_events) {
+            if (categoriesWithEvents.disabled && !categoriesWithSubcategories.disabled && category.has_events && category.deep_category_count === 0) {
```

Comment:
`category.has_events && category.deep_category_count === 0` - that's impossible, you cannot have events but a deep event count of 0